### DISCUSSION TITLE
Add emailadress for European GPSR law

### DISCUSSION THREAD

**Discussion by venditdevs** [PrestaShop/PrestaShop#37651] [Ideas]
### The problem

There's a new rule present in the GPSR law in Europe from the 13th of december 2024. You need to show the data of the supplier (the legally responsible party) when you sell/distribute products.

One of the mandatory information is the e-mailadress of tat party. Prestashop does not have a e-mailaddress field for the suppliers

### The solution

Add a field to fill in the e-mailaddress on a supplier

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by Progi1984** [comment]
@venditdevs You talk about https://en.wikipedia.org/wiki/Regulation_(EU)_2023/988 ?

**Reply by venditdevs** [reply]
> @venditdevs You talk about https://en.wikipedia.org/wiki/Regulation_(EU)_2023/988 ?

Yes, that is the GPSR law

**Comment by Hlavtox** [comment]
@venditdevs I will be resolving this by adding a contact information to manufacturer description. No core mods needed then. :-)
